content/contribute.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Contribute to PyBaMM
+---
+
+If you'd like to contribute to PyBaMM whether via reporting issues and bugs, updating the documentation, or adding new features (thanks!), please have a look at the [Contributing Guidelines](https://github.com/pybamm-team/PyBaMM/blob/develop/CONTRIBUTING.md).
+
+{{< include-md "outline.md" >}}
+
+{{< include-md "donate.md" >}}
```

Comment:
The content after the "For sponsors" heading looks a bit weird - https://deploy-preview-22--pybamm-developer-preview.netlify.app/contribute/

Reply:
Yes, there is some overlap within the contents of the project document and the donations page, and I'm not sure how to proceed – I marked this section as partially complete. I have included both for now so that further discussion can happen on this thread

Reply:
Moved donations to a separate page for now, makes it look neater for the time being